### ISSUE TITLE
damage meters of inventory items showing through menu

### ISSUE THREAD

**Issue by jshipley** [Rearth/Oritech#74]
### Issue description

When items have a colored bar on them, the colored bar is showing through the owo menus.
![Screenshot 2024-08-19 214535](https://github.com/user-attachments/assets/10b0359f-9132-4219-8cad-7b947c91da26)

### Steps to reproduce

Use a tool until it's no longer at 100%. Open a menu that covers the inventory (like the addons menu for a machine).

### Minecraft version

1.21 (Latest)

### Fabric loader version

0.16.0

### Fabric API version

0.100.8+1.21

### Oritech version

0.8.1

**Comment by Rearth** [comment]
Yeah I noticed this issue too when creating the UI. However I have no idea how to fix this. I played around with the z values a lot, and any major increases resulted in completely breaking the rendering

**Comment by jshipley** [comment]
Just collecting some more data to make this easier to look at later:

I just ran into this again with tooltips. In the arcane catalyst, the item tooltip looks fine:

![image](https://github.com/user-attachments/assets/138dda62-19a7-4e96-a1e2-dc94429e94c9)

but the UI tooltip lets the damage meter show through:

![image](https://github.com/user-attachments/assets/a009d4cb-30ac-4a72-b79f-b8e6c9ade3e2)

**Comment by Nova-Katt** [comment]
I can add that I've had the same issue in minecraft 1.21 with fabric 0.16.5, and also minecraft 1.21.1 with fabric 0.16.7.
So whatver the issue is, it spans multiple versions. :)

**Comment by moonfather1** [comment]
> Yeah I noticed this issue too when creating the UI. However I have no idea how to fix this. I played around with the z values a lot, and any major increases resulted in completely breaking the rendering

z isnt the problem, enabled slots cause more than just visual problems. you need to disable the slots (client side) and not render them.

**Comment by Rearth** [comment]
This has been fixed in the latest release.
